### PR TITLE
ARROW-160 Consolidate benchmarks

### PR DESCRIPTION
**Pull request by juliusgeo**

### CODE REVIEW

**Comment by blink1073** [comment]
Can you please add docs under `docs/source/developer` for how to run the benchmarks locally?

**Comment by juliusgeo** [comment]
@blink1073 Done!

File: bindings/python/docs/source/developer/benchmarks.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Running Benchmarks
+==================
+
+.. highlight:: bash
+
+System Requirements
+-------------------
+
+To run the benchmarks, you need the `asv<https://pypi.org/project/asv/>`_ package,
```
Comment (ShaneHarvey) [inline bindings/python/docs/source/developer/benchmarks.rst:9]: This link doesn't render correctly.

Comment (juliusgeo) [reply bindings/python/docs/source/developer/benchmarks.rst]: Done.

**Review by blink1073 — APPROVED** [review]
LGTM!

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**